roles/matrix-bridge-mautrix-discord/templates/systemd/matrix-mautrix-discord.service.j2

```diff
@@ -0,0 +1,43 @@
+#jinja2: lstrip_blocks: "True"
+[Unit]
+Description=Matrix Mautrix Discord bridge
+{% for service in matrix_mautrix_discord_systemd_required_services_list %}
+Requires={{ service }}
+After={{ service }}
+{% endfor %}
+{% for service in matrix_mautrix_discord_systemd_wanted_services_list %}
+Wants={{ service }}
+{% endfor %}
+DefaultDependencies=no
+
+[Service]
+Type=simple
+Environment="HOME={{ matrix_systemd_unit_home_path }}"
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} kill matrix-mautrix-discord 2>/dev/null || true'
+ExecStartPre=-{{ matrix_host_command_sh }} -c '{{ matrix_host_command_docker }} rm matrix-mautrix-discord 2>/dev/null || true'
+
+# Intentional delay, so that the homeserver (we likely depend on) can manage to start.
+ExecStartPre={{ matrix_host_command_sleep }} 5
+
+ExecStart={{ matrix_host_command_docker }} run --rm --name matrix-mautrix-discord \
+			--log-driver=none \
+			--user={{ matrix_user_uid }}:{{ matrix_user_gid }} \
+			--cap-drop=ALL \
+			--network={{ matrix_docker_network }} \
+			-v {{ matrix_mautrix_discord_config_path }}:/config:z \
+			-v {{ matrix_mautrix_discord_data_path }}:/data:z \
```

Comment:
```suggestion
			--mount type=bind,src={{ matrix_mautrix_discord_config_path }},dst=/config,ro \
			--mount type=bind,src={{ matrix_mautrix_discord_data_path }},dst=/data \
```

Hopefully a read-only `/config` directory is okay. Certain mautrix bridges try to update the configuration on startup and include a `--no-update` command line flag to make them not do that.